### ISSUE TITLE
Published sims improperly zoom

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/joist#671]
Found during https://github.com/phetsims/QA/issues/559.
When Molecules and Light was republished recently a proper method for zooming in and out was made for simulations. I noticed that the Charges and Fields sim being tested for maintenance release had a buggy zoom behavior that it should not possess (Shas is before the zoom change in master). I was able to replicate this behavior on some older sims on the published website (published Charges and Fields and Acid Base Solutions). Newer sims (Build a Molecule, Curve Fitting) did not have this behavior. Did not seem to happen on master with sims that had zoom off (Natural Selection).

Behavior also seemed a bit dependent on platform. Bugs were seen on Mac Safari, Mac Chrome, and Win 10 Chrome. It was not seen on Win 10 Firefox. @ariel-phet let me know if you want me to look at other platforms or check any other simulations.

Behavior on Chrome:
1. Pinch to zoom zoomed in and out more or less how it would post MaL (nothing is meant to happen)
2. If zoomed in, scrolling can pan the screen, but you can't use the mouse to drag it around.
3. Cmnd+0 will return to proper size.
4. Cmnd+(+/-) dos not do anything (this is the expected behavior)

Behavior on Safari:
1. Pinch to zoom will zoom in or out, but how much or even if you get a response is inconsistent
2. The zoomed in or out sim may be off center, smaller than normal size, or invisible, or it may look like a zoomed in sim should.
3. Panning did not seem to work
4. Cmnd+0 will return to proper size.
5. Cmnd+(+/-) dos not do anything (this is the expected behavior)

Assigning @ariel-phet for priority, @jessegreenberg due to familiarity with zooming behavior, and @jonathanolson by request.

**Comment by jessegreenberg** [comment]
@KatieWoe can you please confirm that by "Pinch to zoom" you mean on a trackpad? Pinching and zooming on a touch screen does nothing, correct?

**Comment by KatieWoe** [comment]
Yes, on trackpad (Mac I'm using doesn't have a touchscreen)

**Comment by KatieWoe** [comment]
On safari it may be easier if you start trying to zoom while the simulation is still loading. This seems to be required for the the iPadOS 14 Safari.

**Comment by jessegreenberg** [comment]
OK thanks, that makes sense. I suspect that this is how it has always been for HTML5 sims with trackpads. Trackpad zooming works different than other browser zooming. For example, if you zoom in with a trackpad you will not see the browser's zoom indicator
![image](https://user-images.githubusercontent.com/6396244/95919545-c7410e00-0d7b-11eb-8d1a-fda469dc4a44.png)

> Behavior on Chrome:

This list is what I would expect for sims without PhET's pan and zoom feature and is the zoom behavior for all other websites that I try it on.

> Behavior on Safari:
>> The zoomed in or out sim may be off center, smaller than normal size, or invisible, or it may look like a zoomed in sim should.
>> Panning did not seem to work

This is weird and we may want to do something about it. New sims with PhET's pan and zoom feature will not have this problem. Sims without pan and zoom (older sims and sims that disable the feature) will have this problem. We can either disable trackpad zoom in this case, or investigate why it isn't behaving as expected. Disabling would be easiest, we already disable it when pan/zoom is enabled to take over for our own implementation. Since this is not an issue for sims with PhET's pan and zoom and since it has never been reported to us by users it may not be worth the time to fix.

@ariel-phet what do you think?

> On safari it may be easier if you start trying to zoom while the simulation is still loading. This seems to be required for the the iPadOS 14 Safari.

This part *is* related to touch screen input, and it sounds like we need to prevent default behavior on the splash screen if possible.

**Comment by ariel-phet** [comment]
@jessegreenberg @KatieWoe 

I can reproduce this odd behavior on Charges and Fields (published version), if zooming in while the sim loads using pinch to zoom on iPad iOS 14.

However, pinching to zoom on the loading screen seems like "edge case" behavior. Importantly if I pinch to zoom on the website, and then launch the sim, it launches normally.

As @jessegreenberg points out, the behavior seen with track pads and such has never been reported by users, so I do not think it is worth the time to fix. I also do not think the splash screen behavior is worth retroactively fixing (again, quite an edge case).

@jessegreenberg checking published Molecules and Light I do see quite odd behavior if I pinch to zoom on the splash screen. although I do not think we need to make any retroactive fixes, and this is an edge case, but it would be good to investigate a fix since we are aware of the issue (and on a slow loading device it is conceivable a student my play with pinch to zoom when that becomes a regular expectation in our sims). 

So basically, no batch maintenance releases or such necessary, but going forward it might be good to fix up the splash screen behavior (and if easy potentially patch MAL).

**Comment by jessegreenberg** [comment]
The splash screen doesn't use scenery or the AnimatedPanZoomListener, so it would be much easier if we can just disable pan and zoom on the splash screen rather than supporting zoom there, then trying to get the zoom to carry over to the sim. Over slack I confirmed with @ariel-phet that this is an OK solution. Since that is the only action item for this issue, I am going to make a new issue to disable pinch and zoom on the splash screen and close this one.

**Comment by jessegreenberg** [comment]
Going to pick up in #672, closing.
